validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -70,7 +70,7 @@
 public class ValidatorClientService extends Service {
 
   private static final Logger LOG = LogManager.getLogger();
-  private static final Duration DOPPELGANGER_DETECTOR_CHECK_DELAY = Duration.ofSeconds(5);
+  private static final Duration DOPPELGANGER_DETECTOR_CHECK_DELAY = Duration.ofSeconds(12);
```

Comment:
Is it any sense to tie it to the slot duration? (remember gnosis have 5 seconds slots)

Reply:
It's actually a tradeoff between Dg detection speed and effectiveness.
5s is too short for 12s slots since it doesn't really make sense to check after at least one slot since we're almost sure there have been no new attestations and hence no eventual active validator caching.
12s is more suitable for both 12s slots and shorter slots...